python/dgl/data/sbm.py

```diff
@@ -49,9 +53,12 @@ def sbm(n_blocks, block_size, p, q, rng=None):
     adj = sp.sparse.triu(a) + sp.sparse.triu(a, 1).transpose()
     return adj
 
-class SBMMixture:
-    """ Symmetric Stochastic Block Model Mixture
-    Please refer to Appendix C of "Supervised Community Detection with Hierarchical Graph Neural Networks" (https://arxiv.org/abs/1705.08415) for details.
+
+class SBMMixtureDataset(object):
```

Comment:
Is this a self-contained dataset that do not need download data?
You should derived from DGLDataset at least.